Update the message sequence EIP (#743)
wso2/docs-mi: en/docs/learn/enterprise-integration-patterns/message-construction/message-sequence.md

![Message sequence size]({{base_path}}/assets/img/learn/enterprise-integration-patterns/message-construction/message-sequence-size.gif)

## How WSO2 Micro Integrator implements the EIP

> When the sent request is large, the Micro Integrator by default breaks it into smaller chunks. In order to maintain the consistency of the message, each chunk is then mapped with a sequence identity number so that its sequential order is not lost.

Reviewer: ```suggestion
When a large request is sent, the Micro Integrator, by default breaks it into smaller chunks. To maintain the consistency of the message, each chunk is then mapped with a sequence identifier number to preserve its sequential order.
```
